Explorer/Assets/Scripts/CrdtEcsBridge/JsModulesImplementation/PortableExperiencesApiImplementation.cs

```diff
@@ -0,0 +1,42 @@
+using Cysharp.Threading.Tasks;
+using PortableExperiences.Controller;
+using SceneRunner.Scene;
+using SceneRuntime.Apis.Modules.PortableExperiencesApi;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace CrdtEcsBridge.PortableExperiencesApi
+{
+    public class PortableExperiencesApiImplementation : IPortableExperiencesApi
```

Comment:
This class looks like an extra-step. Do we really need it? Could `PortableExperiencesApiWrapper` interact with `IportableExperiencesController` directly?

Reply:
Well, tbh, it is how all APIs are implemented for communication with js side AFAIK?
we create a wrapper for the js calls and then wrapper calls the APIs that actually have the logic?
Using the IJsApiWrapper/JsApiWrapperBase classes... But let me know if it seems wrong how I did it thou xD

Reply:
It feels wrong to me since because we are adding extra complexity to the solution without needed. The feature is already encapsulated into its own class besides the js wrapper. Here is an example that doesnt include an api, since the repository itself was enough to make it work: https://github.com/decentraland/unity-explorer/blob/27eb2dca146a61ec21c937b96e9e57e443f3d947/Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/UserIdentityApi/UserIdentityApiWrapper.cs#L49

Reply:
TBH this approach seems much more clean :D I didnt know we had it xD
I prefer it at having the additional classes which make dependencies and assemblies a pain in the ass to manage.
I implemented your suggestion! looks way better 💪